source/pervasive/nonlinear_arith/div.rs

```diff
@@ -0,0 +1,1088 @@
+//! Lemmas for division.
+
+use builtin::*;
+use builtin_macros::*;
+#[allow(unused_imports)]
+use crate::calc_macro::*;
+
+verus! {
+
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::div_internals::{div_recursive, lemma_div_induction_auto, div_auto, div_pos, lemma_div_auto};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::div_internals_nonlinear as DivINL;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mod_internals::{lemma_mod_auto, mod_recursive};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mod_internals_nonlinear as ModINL;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mul_internals::{lemma_mul_induction_auto, lemma_mul_auto, mul_auto, lemma_mul_induction};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::mul::*;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::general_internals::{is_le};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::modulus::*;
+
+/*****************************************************************************
+ * Division:
+ *****************************************************************************/
+
+/// the common syntax of division gives the same quotient as performing division through recursion
```

Comment:
Starting document comments with a capital letter is good; you can also use markdown syntax (for: `/`). Also, using the square bracket syntax that I'm suggesting will allow the documentation tool to connect the two things up with a hyperlink automatically.

```suggestion
/// Division using `/` is equivalent to division through [`div_recursive`]
```

I'll let you fix up other locations similar to this in the same way